### ISSUE TITLE
Original XBOX 360 USB controller not detected.

### ISSUE THREAD

**Issue by dipunm** [360Controller/360Controller#553]
I have tried the following steps:
- unplug and replug the controller
- enable driver from Advanced tab (not sure why it was disabled)
- disable and re-enable driver.
- uninstall drivers and install 0.16.4 (because my controller is old and 0.16.5 says it supports only new devices)

Here is my driver report:

![image](https://user-images.githubusercontent.com/5099980/32693607-c51c90da-c725-11e7-8c8c-ed1d4c0cc1ff.png)

and my OS:

![image](https://user-images.githubusercontent.com/5099980/32693612-d83309ec-c725-11e7-918d-5bc367ecf958.png)

**Comment by DoctorAngelo** [comment]
I have macOS sierra high and xbox 360 controller but it doesn't work. Please help.

**Comment by rghedin** [comment]
Same here.

**Comment by sterpe** [comment]
FWIW, I have been able to get an [X-Arcade Stick w/PS2->Xbox360](https://shop.xgaming.com/collections/adapters/products/x-arcade-xbox-360-adapter) adapter working on High Sierra (10.13.1) via USB 2.0.  Installed 0.16.5 `.dmg`, approved drivers signed by (Drew???) in Security Preferences, reboot, enable drivers in System Preferences (have to do this everytime, for some reason).

@dipunm My thought would be first do upgrade to 10.13.1.  Also maybe try to connect to older USB 2.0 hub and then into machine, if you have one lying around, rather plugging controller directly into builtin USB3.

![screen shot 2017-11-19 at 15 07 45](https://user-images.githubusercontent.com/678398/32996744-b5a4686e-cd3b-11e7-9926-1c1ba44b4284.png)
![screen shot 2017-11-19 at 15 07 35](https://user-images.githubusercontent.com/678398/32996746-b8e9d022-cd3b-11e7-9f14-cf3e205e621c.png)
![screen shot 2017-11-19 at 15 07 18](https://user-images.githubusercontent.com/678398/32996749-bc471018-cd3b-11e7-9705-fe5cfb8e466c.png)

**Comment by dipunm** [comment]
I got it working thanks to @sterpe. I updated my OS, and the application. The one thing that I think was key was the privacy settings step. For others: open System Preferences and go to "Security & Privacy"; then go to the general tab.

![image](https://user-images.githubusercontent.com/5099980/33148013-0d46dd08-cfc2-11e7-9468-a130a83647a5.png)

Theres a button to allow installation of drivers from Drew. Allow it, and restart. Voilla!
Even works without a USB hub for me. :) Thanks.
